### PR TITLE
Removed closeAllConnections from tests (fixes #135)

### LINKED ISSUE #135: uptasticsearch broken on CRAN
Got one of those "your package is suddenly broken on CRAN fix it ASAP or we're taking it down" emails from Kurt Hornik last night.

You can see the results here: https://cran.r-project.org/web/checks/check_results_uptasticsearch.html

The relevant bit of the log is:

```
 > testthat::test_check('uptasticsearch')
     ── 1. Error: (unknown) (@test-parse_date_time.R#180) ──────────────────────────
     there is no connection 3
     1: closeAllConnections() at testthat/test-parse_date_time.R:180
     2: close(getConnection(set[i]))
     3: getConnection(set[i])
```

We haven't pushed a new release June 2018, so that means that either they added new environments they test in or there was a breaking change in `closeAllConnections()`.

I think we can just remove `closeAllConnections()` calls from our tests.

### PR DESCRIPTION
**Pull request by jameslamb**
This should get CRAN off our backs. If you approve I'll cut a release and push it, but let's keep the PR open until we get accepted.

### CODE REVIEW

**Comment by jameslamb** [comment]
I had to add `testthat`, `rmarkdown`, and `knitr` to the build. Think they must have made the default Travis env for R more lightweight.

**Comment by jameslamb** [comment]
oh I know why CRAN failed on my submission! I forgot we moved the test data out to a shared folder. I'll update `CONTRIBUTING.md` and add a build thing in the `Makefile`

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+To report bugs, request features, or ask questions about the structure of the code, please [open an issue](https://github.com/UptakeOpenSource/uptasticsearch/issues).
+
+### Bug Reports
+
+If you are reporting a bug, please describe as many as possible of the following items in your issue:
+
+- your operating system (type and version)
+- your version of R
+- your version of `uptasticsearcht`
```
Comment (austin3dickey) [inline CONTRIBUTING.md]:
```suggestion
- your version of `uptasticsearch`
```

**Comment by jameslamb** [comment]
The build failure should be fixed when we merge #138 and rebase this (I'll do that soon). Couple things I had to fix that are unrelated to this and just related to the fact that Travis has changed under our feet in the last couple months

**Comment by jameslamb** [comment]
@austin3dickey I updated `cran-comments.md` and also updated the `pkgdown` site (seemed like the right thing to do for a release commit)

**Comment by austin3dickey** [comment]
looks great!

**Pull request merged**